Modules/Scripted/EditorLib/HelperBox.py

```diff
@@ -165,19 +165,17 @@ def setVolumes(self,masterVolume,mergeVolume):
 
   def setMasterVolume(self,masterVolume):
     """select merge volume"""
+    if isinstance(masterVolume, basestring):
+      masterVolume = slicer.mrmlScene.GetNodeByID(masterVolume)
     self.masterSelector.setCurrentNode( masterVolume )
-    self.select()
```

Comment:
Agreed. 

On a similar topic, I wonder if it would make more sense to explicitly created the label map if none is available instead of trying to lazily create it. It means that we would check if a merge volume is needed in the `Editor/enter()` function.